### ISSUE TITLE
_get_keyfile_() function is not generating dynamic directory path

### ISSUE THREAD

**Issue by harunurkst** [Primechain/blockchain-ekyc-sawtooth#3]
When I running bash category.sh, It shows error   

`Exception: Failed to read private key /home/harun/sawtooth/sawtooth-ekyc-member/jsclient/keys/categories.priv: [Errno 2] No such file or directory: '/home/harun/sawtooth/sawtooth-ekyc-member/jsclient/keys/categories.priv'`   

I noticed that, In simplewallet_cli.py, _get_keyfile() and _get_pubkeyfile() is generating static path. It should generate a dynamic KEY path. I changed the path as my local directory and it works fine.

**Comment by rahul0812** [comment]
Yes, as of now it generates static path. Thank you for the suggestion, we will address this issue.
